plugins/callback/print_task.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Max Mitschke <maxmitschke@fastmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: print_task
+type: aggregate
+short_description: Prints playbook task snippet to job output
+description:
+  - This plugin prints the currently executing playbook task to the job output.
+version_added: 10.7.0
+requirements:
+  - enable in configuration
+'''
+
+
+import yaml
```

Comment:
I would advise against using 'plain' yaml, i would try using the CSafeLoader first (speed of C lib), fallback to SafeLoader (slower python) and avoid other loaders. See core's use of yaml lib when in doubt.

Reply:
Just updated the code to do this.